other-workflows.tex

```diff
@@ -1,3 +1,13 @@
-\subsubsection{Comparison with other workflows}
+\subsubsection{Comparison With Other Workflows}\label{sec:other_git_workflows}
+
+The LSST workflow is very similar to GitHub Flow \cite{GitHubFlow}, differing primarily in how merging is handled.
+GitHub Flow does not include a rebasing step because it does not require the commit history to be linearizable.
+Instead, in GitHub Flow, pull requests are considered the primary record of a repository's history.
+While the LSST workflow preserves both pull requests and feature branches, the primary source of a repository's history is the Jira tickets and the commits themselves.
+
+The LSST workflow does not have the separate integration and release branches characteristic of the other major Git workflow, Gitflow \cite{Gitflow}.
+Because package development is not tied directly to the release schedule [TODO: reword so that it allows for project milestones],
+and because LSST does not provide support for any but the latest release of the data management system,
+managed release branches are an unnecessary complication in the LSST workflow.
```

Comment:
Although, as we approach commissioning we may well start to adopt release branches. We do currently use release branches for the major six monthly releases but they have very few commits on them and only those needed to fix major problems during the release process.

Reply:
That is typically what release branches are supposed to look like.

In any case, my knowledge of LSST's release process is minimal (and apparently wrong), so I don't think I can write that bit. I can add a TODO for somebody else to do it.